packages/ui-components/src/illustrations/illustration-component.hbs

```diff
@@ -21,6 +21,7 @@ export class {{className}} implements ICustomCss {
 				<svg
 					part="illustration"
 					xmlns="http://www.w3.org/2000/svg"
+					style=\{{ "display": "flex" }}
```

Comment:
You have an illustration class where you can put this style so you don't need to inline it.